src/main/resources/static/api.yaml

```diff
@@ -1342,14 +1342,14 @@ paths:
               $ref: '#/components/schemas/RecoverCmIdRequest'
       responses:
         '200':
-          description: Found a patient
+          description: Details of the session and otp are sent in response. otpMedium defines verified identifier where it would be sent to. Its value is mentioned in otpMediumValue. OTP has an expiry time in minutes which is also sent.
```

Comment:
please rename the endpoint as "/patients/profile/recovery-init"

Reply:
yeah will do